components/export/api/src/checkstyle/checkstyle-suppressions.xml

```diff
@@ -24,4 +24,6 @@
 <suppressions>
   <suppress checks=".*" files="DataToCellConverter.java"/>
   <suppress checks="CyclomaticComplexity" files="Styler.java" lines="155, 209"/>
+  <suppress checks="ClassDataAbstractionCoupling" files="GeneHGNCNomenclature.java" lines="77, 341"/>
```

Comment:
These are in the wrong file. Each module can have its own `checkstyle-suppressions.xml` file, so you must add a new file in the `components/ontology-access/hgnc/` module and [enable it](https://github.com/phenotips/phenotips/blob/master/components/export/api/pom.xml#L33) in the pom.